### PR TITLE
Feature/rado 6

### PR DESCRIPTION
**Pull request by sandrosc**

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -17,5 +17,8 @@
       "*": ["node_modules/*", "src/types/*"]
     },
     "lib": ["es2017", "dom"]
+  },
+  "angularCompilerOptions": {
+    "enableIvy": true
```
Comment (joelfisch) [inline tsconfig.json:22]: Why did you enable ivy?

File: package.json [hunk 2]
```diff
@@ -30,6 +30,7 @@
     "angular-cli-ghpages": "^1.0.0-rc.1",
     "antlr4": "^4.7.2",
     "core-js": "^2.6.2",
+    "date-format": "^3.0.0",
```
Comment (joelfisch) [inline package.json:33]: Why do you use a library for the time format even though you format everything else yourself?

**Review by joelfisch — APPROVED** [review]

File: tsconfig.json (hunk 1 above)
Comment (sandrosc) [reply tsconfig.json:22]: so angular devtools work: https://chrome.google.com/webstore/detail/angular-devtools/ienfalfjdbdpebioblfackkekamfmbnh

Comment (joelfisch) [reply tsconfig.json:22]: 👍

File: package.json (hunk 2 above)
Comment (sandrosc) [reply package.json:33]: good point, removed.

**Pull request merged**